docs/include/content.tex

```diff
@@ -235,7 +219,75 @@ \chapter{How to set up Quarkus for remote development}
+            \item Before you are able to create and run the container, you have to create a seperate network for your container, for example to connect a database (\ref{sec:database}). \label{item:network}
+            \begin{lstlisting}[language=bash]
+docker network create --driver bridge quarkus-net
+            \end{lstlisting}
+            \item After you have created your image, you can create a container which is attached to the currently created network:
+            \begin{lstlisting}[language=bash, breaklines=true]
+docker run --network quarkus-net --name quarkus -it quarkus_image
+            \end{lstlisting} 
+        \end{enumerate}
+        The installation of the gpg keys to sign your commits has to be done manually as well but the project provides a generation script \textit{gen-key-script}. Attach to the Quarkus container and follow these steps: 
+        \begin{enumerate}
+            \item Before executing the generation script you have to replace the \textit{REPLACE-ME}-tokens with your own credentials. Do not forget your passphrase, you will need it for signing your commits. Afterwards you can run the script: 
+            \begin{lstlisting}[language=bash]
+gpg --batch --gen-key gen-key-script
+            \end{lstlisting}
+            \item Now you have to tell git and GitHub your signing key. Follow the steps described from \ref{itm:gpg_key_start} to \ref{itm:gpg_key_end} in section \ref{sec:quarkus_remote}.
+        \end{enumerate}
+        
+
+    \section{Attach Visual Studio Code to Docker Container}\label{sec:attach}
+    Open VSCode and navigate to the Docker-symbol in the vertical-left bar as shown in figure \ref{fig:attach_vscode}. \\
+    When Docker Desktop is running, you will see all your containers. Maybe you have to start the container first. \\
+    Peform a right click on the respective container and select \glqq Attach Visual Studio Code\grqq. 
+    \begin{figure}[h]
+        \centering
+        \includegraphics[scale=0.7]{images/attach_VSCode.png}
+        \caption{Attach Docker Container to VSCode}
+        \label{fig:attach_vscode}
+    \end{figure}
+    
+    Afterwards the container will be attached to VSCode, which can be seen by the green information in the bottom left corner. In figure \ref{fig:container_attached} you can see an example. The text may vary, depending on your image and container name.     
+    
+    \begin{figure}[h]
+        \centering
+        \includegraphics{images/container_attached.png}
+        \caption{Docker Container is attched}
+        \label{fig:container_attached}
+    \end{figure}
+
+    \section{How to set up a database container}\label{sec:database}
+        In this section you will get some information how to create a database within a container and connect it to your quarkus container.  
+        \begin{enumerate}
+            \item Create and run a PostgreSQL database. The quarkus-net network has to be already created (look at \ref{item:network}):
```

Comment:
The link (look at ...) does not work. The reader of your pdf gets a "2" displayed.